components/transfer/Transfer.vue

```diff
@@ -164,13 +175,17 @@
                 min="0"
                 icon-right="usd"
                 icon-right-class="has-text-grey"
+                data-testid="transfer-input-amount-usd"
                 @focus="onAmountFieldFocus(destinationAddress, 'usd')"
                 @input="onUsdFieldChange(destinationAddress)" />
               <a
                 v-if="!isMobile && targetAddresses.length > 1"
                 class="is-flex"
                 @click="deleteAddress(index)">
-                <NeoIcon class="p-3" icon="trash" />
+                <NeoIcon
+                  class="p-3"
+                  icon="trash"
+                  data-testid="transfer-remove-recipient" />
```

Comment:
not actionable

Reply:
> not actionable

can you explain in noob terms? did not get it

Reply:
You add a test on Icon, I don't find it usefull, I would add it on a button

Reply:
> You add a test on Icon, I don't find it usefull, I would add it on a button

ty, added it to the <a> tag above, since there's no button near it